bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2BridgeHandler.java

```diff
@@ -486,7 +486,15 @@ private void initializeAssets() {
 
             String applicationKey = config.applicationKey;
             applicationKey = Objects.nonNull(applicationKey) ? applicationKey : "";
-            clip2Bridge = new Clip2Bridge(httpClientFactory, this, ipAddress, applicationKey);
+
+            try {
+                clip2Bridge = new Clip2Bridge(httpClientFactory, this, ipAddress, applicationKey);
+            } catch (ApiException e) {
+                logger.trace("initializeAssets() communication error on '{}'", ipAddress, e);
```

Comment:
Minor: This is exception from the constructor, not `initializeAssets()`. Also, strictly speaking, this is not a communication error, but failure to start a component (HttpClient).

Reply:
> how well tested are these fixes

All the changes are backported to my v3.4.x operative system and therefore live tested.